apps/ledger-live-desktop/tests/specs/manager/customImage.spec.ts

```diff
@@ -0,0 +1,113 @@
+import test from "../../fixtures/common";
+import { expect } from "@playwright/test";
+import { ManagerPage } from "../../models/ManagerPage";
+import { DeviceAction } from "../../models/DeviceAction";
+import { Layout } from "../../models/Layout";
+import { CustomImageDrawer } from "../../models/CustomImageDrawer";
+import { DeviceModelId } from "@ledgerhq/devices";
+
+test.use({ userdata: "skip-onboarding", featureFlags: { customImage: { enabled: true } } });
+
+test("Custom image", async ({ page }) => {
+  const managerPage = new ManagerPage(page);
+  const deviceAction = new DeviceAction(page);
+  const customImageDrawer = new CustomImageDrawer(page);
+  const layout = new Layout(page);
+
+  const container = customImageDrawer.container;
+
+  await test.step("Access manager", async () => {
+    await layout.goToManager();
+    await deviceAction.accessManager("", "", DeviceModelId.nanoFTS);
+    await managerPage.firmwareUpdateButton.waitFor({ state: "visible" });
```

Comment:
should it be `managerPage.customImageButton` here ?

Reply:
I don't know what you are talking about

![dd0](https://user-images.githubusercontent.com/91890529/192574784-c3307311-518f-4495-8800-02f4ce70c2ba.png)

Reply:
Why are we waiting for the `firmwareUpdateButton` to be visible here, and not `customImageButton` ?